src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -246,88 +444,164 @@ public virtual async Task<Response> DeleteAsync(string key, string label = defau
         /// <param name="key">The primary identifier of a configuration setting.</param>
         /// <param name="label">The value used to group configuration settings.</param>
         /// <param name="acceptDateTime">The setting will be retrieved exactly as it existed at the provided time.</param>
-        /// <param name="cancellation">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        public virtual async Task<Response<ConfigurationSetting>> GetAsync(string key, string label = default, DateTimeOffset acceptDateTime = default, CancellationToken cancellation = default)
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        public virtual Response<ConfigurationSetting> Get(string key, string label = default, DateTimeOffset acceptDateTime = default, CancellationToken cancellationToken = default)
```

Comment:
should we add test for this sync methods? would there be value on having them as a client library test?

Reply:
Yes, I'm going to handle this as part of followup PR, this one is becoming large.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/6020